ci/build-ios_client.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+set -e -u -x -o pipefail
+
+cd "$(dirname "$0")/../"
+
+source .shared-ci/scripts/pinned-tools.sh
+source .shared-ci/scripts/profiling.sh
+
+UNITY_PROJECT_DIR="$(pwd)/workers/unity"
+
+markStartOfBlock "$0"
+
+markStartOfBlock "Building iOSClient"
+
+TARGET=${1:-local}
+
+# Back-compat: we need to run this to generate the bridge configurations.
+spatial build build-config IosClient
```

Comment:
do we need to run this for the build step? this is only needed when running a deployment

Reply:
ah just saw, that we also did this in the other scripts. not sure why though

Reply:
I think this was before we built the worker configs during the build (actually does this happen from the CLI? If not we can add it there and get rid of these) 

The idea being the build script would build everything required to run a worker